src/AzureClient/Extensions.cs

```diff
@@ -22,5 +30,96 @@ public static void AddAzureClient(this IServiceCollection services)
         {
             services.AddSingleton<IAzureClient, AzureClient>();
         }
+
+        /// <summary>
+        ///      Encapsulates a given <c>AzureClientError</c> as the result of an execution.
```

Comment:
Minor, but `<see cref="AzureClientError" />` can be nicer than `<c></c>` alone for generating links.

Reply:
Thanks, I forgot about that syntax.

---
In reply to: [427470952](https://github.com/microsoft/iqsharp/pull/144#discussion_r427470952) [](ancestors = 427470952)